libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Analytics/SFSDKSalesforceAnalyticsManager.m

```diff
@@ -44,6 +44,7 @@
 static NSString * const kEventStoresDirectory = @"event_stores";
 static NSString * const kEventStoreEncryptionKeyLabel = @"com.salesforce.eventStore.encryptionKey";
 static NSString * const kAnalyticsOnOffKey = @"ailtn_enabled";
+static NSString * const kSFAppFeatureAiltnEnabled = @"AI";
```

Comment:
That's true. I'll merge this for now if that's okay and leave it to @ivanbogdanov, since he's working on adding app features.